content/docs/components/serving/kfserving.md

```diff
@@ -0,0 +1,46 @@
++++
+title = "KFServing"
+description = "Using KFServing for serving Models"
+weight = 2
++++
+
+## KFServing with Kubeflow
+
+KFServing can be installed with Kubeflow v0.7, and KFServing Kustomize install files are [located in the manifests repo](https://github.com/kubeflow/manifests/tree/master/kfserving).
```

Comment:
s/install/installation

The noun is "installation" (or "installer") and the verb is "install". It's confusing for readers to see "install" used as a noun, and such usage can lead to problems in translating from English to other languages.